packages/oca/src/types/branding/BrandingOverlay.ts

```diff
@@ -19,7 +19,8 @@ export default class BrandingOverlay extends BaseOverlay {
     this.logo = overlay.logo
     this.#background_image = overlay.background_image
     this.#background_image_slice = overlay.background_image_slice
-    this.#primary_background_color = overlay.primary_background_color ?? generateColor(credentialDefinitionId)
+    this.#primary_background_color =
+      overlay.primary_background_color ?? generateColor(credentialDefinitionId ?? 'default')
```

Comment:
This should resolve to a valid color or an empty string. Where would `'default'` be used?

Reply:
When this overlay is used on a proof. The proof screen doesn't provide a cred-def-id, so this ensures that a color will always be set

Reply:
Hmm. I think that logic should stay separate from this for now, a `default` is not really compatible with the specification requiring a Hex code or RGB color